test/model/model_test.rb

```diff
@@ -126,6 +134,13 @@ class Event
         assert !Event.bucket.exist?(@event.id)
       end
 
+      test 'delete with quorum options' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.